bigtable_rs/src/bigtable.rs

```diff
@@ -493,6 +494,32 @@ impl BigTable {
         decode_read_rows_response(self.timeout.as_ref(), response).await
     }
 
+    /// Streaming support for `read_rows` method
+    pub async fn stream_rows<'a>(
+        &'a mut self,
+        request: ReadRowsRequest,
+    ) -> Result<impl Stream<Item = Result<(RowKey, Vec<RowCell>)>>> {
+        let response = self.client.read_rows(request).await?.into_inner();
+        let stream = decode_read_rows_response_stream(response).await;
+        Ok(stream)
+    }
+
+    /// Streaming support for `read_rows_with_prefix` method
+    pub async fn stream_rows_with_prefix<'a>(
+        &'a mut self,
+        mut request: ReadRowsRequest,
+        prefix: Vec<u8>,
+    ) -> Result<impl Stream<Item = Result<(RowKey, Vec<RowCell>)>>> {
+        let row_range = get_row_range_from_prefix(prefix);
+        request.rows = Some(RowSet {
+            row_keys: vec![],
+            row_ranges: vec![row_range],
+        });
+        let response = self.client.read_rows(request).await?.into_inner();
+        let stream = decode_read_rows_response_stream(response).await;
+        Ok(stream)
+    }
```

Comment:
I believe you can safely remove the lifetime specifier, now.

```suggestion
    /// Streaming support for `read_rows` method
    pub async fn stream_rows(
        &mut self,
        request: ReadRowsRequest,
    ) -> Result<impl Stream<Item = Result<(RowKey, Vec<RowCell>)>>> {
        let response = self.client.read_rows(request).await?.into_inner();
        let stream = decode_read_rows_response_stream(response).await;
        Ok(stream)
    }

    /// Streaming support for `read_rows_with_prefix` method
    pub async fn stream_rows_with_prefix(
        &mut self,
        mut request: ReadRowsRequest,
        prefix: Vec<u8>,
    ) -> Result<impl Stream<Item = Result<(RowKey, Vec<RowCell>)>>> {
        let row_range = get_row_range_from_prefix(prefix);
        request.rows = Some(RowSet {
            row_keys: vec![],
            row_ranges: vec![row_range],
        });
        let response = self.client.read_rows(request).await?.into_inner();
        let stream = decode_read_rows_response_stream(response).await;
        Ok(stream)
    }
```

Reply:
ah, yes, thanks!